modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -510,2051 +512,2278 @@ class TFImporter
 
 private:
     void addPermuteLayer(const int* order, const std::string& permName, Pin& inpId);
+
+    typedef std::map<std::string, void (TFImporter::*)(tensorflow::GraphDef&, tensorflow::NodeDef&, LayerParams&)>
+            DispatchMap;
+
+    static DispatchMap dispatch;
+    static DispatchMap initDispatch();
+
+    void convolution        (tensorflow::GraphDef& net, tensorflow::NodeDef& layer, LayerParams& layerParams);
```

Comment:
> `tensorflow::GraphDef& net`

It makes sense to remove this parameter and use field from the class directly (through `this`) instead.

Reply:
In onnx importer there is a diagnostic run and if it is enabled, the other net is used instead, so I left a parameter here to be able to do the same.

Reply:
What is the problem to change field value for that case?

Reply:
https://github.com/opencv/opencv/blob/3e513ee6ab7a23ff5d498e3fb9518665777ed415/modules/dnn/src/onnx/onnx_importer.cpp#L115-L117
Net field is a reference itself, so we could make a self-referential struct, but the reference will become dangling when the object will be moved.